### PR TITLE
Add craigslist

### PR DESCRIPTION
**Pull request by dleshem**
Alexa rank: [150](https://www.alexa.com/siteinfo/craigslist.org)

### CODE REVIEW

**Comment by Carlgo11** [comment]
@dleshem the build failed because your fork is too old. You should probably pull from upstream before you make your next PR

File: _data/other.yml [hunk 1]
```diff
@@ -115,6 +115,12 @@ websites:
       - totp
     doc: https://www.contentful.com/faq/two-factor-authentication/
 
+  - name: Craigslist
+    url: https://craigslist.org/
+    img: craigslist.png
+    twitter: craigslist
+    facebook: Craigslist-74561112489
```
Comment (Carlgo11) [inline _data/other.yml]: Page returns 404 for me.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 1 above)
Comment (dleshem) [reply _data/other.yml]: Twitter: https://twitter.com/craigslist
Facebook: https://www.facebook.com/Craigslist-74561112489

Both work for me.

Comment (kmpoppe) [inline _data/other.yml]:
```suggestion
    facebook: officialcraigslist
```
That page doesn't 404 for me and it looks nothing less official than the twitter account with it's last activity in 2009.
I really wonder if there is any point in adding those social media channels as they don't seem to care about them at all.

Comment (dleshem) [reply _data/other.yml]: I don't mind changing, but the page I linked to has 451,611 likes, while this one has 1,140. Let me know.

Comment (dleshem) [reply _data/other.yml]: You can also try this URL (which is the one Facebook's HTML marks as canonical): https://www.facebook.com/pages/category/Advertising-Marketing/Craigslist-74561112489/

Comment (dleshem) [reply _data/other.yml]: OK, I give up :( Let's go with the Facebook page that has 0.25% the number of likes.

**Pull request closed without merging**